### DISCUSSION TITLE
current version

### DISCUSSION THREAD

**Discussion by gmbh07** [dominikamann/oekofen-pellematic-compact#62] [General]
Hallo,

ich habe heute die Integration über HACS installiert, läuft soweit. Bei mir steht aber unter Einstellungen/Geräte&Dienste/ÖkoFEN/v2.0.0

Habe ich nun die neueste Version installiert oder habe ich hier einen Fehler gemacht?

<img width="1253" alt="Bildschirmfoto 2024-08-07 um 09 04 02" src="https://github.com/user-attachments/assets/6926cf79-0d29-4789-b374-696b5c3a80fb">

**Comment by dominikamann** [comment]
Hi, 

ausschlagend ist das hier. Warum 2.0.0 angezeigt wird ist mir ein Rätsel. Ist jedoch bei mir aber auch so. Ich prüfe das! Danke.

![grafik](https://github.com/user-attachments/assets/3d61d1ac-e944-41f4-9398-34669eb06a85)
